storybookjs/storybook
# Core: Fix IE11 compatibility by using XHR and plain ES5

Issue: #13314

## What I did

This fixes IE11 by using `XMLHttpRequest` instead of `fetch` and avoiding the ES6 object shorthand syntax in the runtime error reporting code.

## How to test

- Is this testable with Jest or Chromatic screenshots? Yes, Chromatic should have picked up the IE11 breakage
- Does this need a new example in the kitchen sink apps? No
- Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.
